README.md

```diff
@@ -77,7 +79,7 @@ Installing on macOS:
 
 - Install dependencies using Homebrew:
   ```
-  brew install autoconf automake libtool bdw-gc boost bison pkg-config
+  brew install autoconf automake libtool bdw-gc boost bison pkg-config protobuf
```

Comment:
We use the `proto3` syntax which requires a protobuf version >= 3.0.0. I think it would be nice to mention this in the README and give instructions.